digital-twin/Samples/service/JdkSample/get-digital-twin/readme.md

```diff
@@ -0,0 +1,66 @@
+# Get Digital Twin Sample
+
+This sample code demonstrates how to check the state of a single digital twin.
+
+## How to run the sample
+
+Note that the samples for Windows and Linux both use Maven, and the below commands work in both windows command line and in
+a linux bash shell.
+
+### Setup environment
+Prepare your platform following the instructions [here][devbox-setup] to install Java and Maven.
+If you want to run these samples in Intellij, follow [these][intellij-setup] instructions to prepare your Intellij settings
+
+### Sample Arguments
+
+In order to run this sample, you must set environment variables for:
+- "IOTHUB_CONNECTION_STRING" : Your IoT Hub's connection string
+- "DEVICE_ID" : The ID of the device to get the twin of
+
+### Other Prerequisites
+In order to run this sample, you will need an IoT Hub. You will also need at least one device registered in this hub so that the sample can lookup the state of that device
+* [Setup Your IoT Hub][lnk-setup-iot-hub]
+
+### Run the sample
+
+From the base digital-twin folder in this repo, run the following command:
+
+```sh
+mvn clean install -DskipTests
+```
```

Comment:
do you need "-DskipTests" for sample run?

Reply:
removed